templates/zerver/help/restrict-user-email-addresses-to-certain-domains.md

```diff
@@ -0,0 +1,32 @@
+# Restrict user email addresses to certain domains
+
+Only administrators can restrict user email addresses to certain domains.
+ This can be done through the administration page.
+
+{!go-to-the.md!} [Organization settings](/#administration/)
+{!admin.md!}
+
+2. Restricting user email addresses to certain domains can be enabled or disabled
+ with the **Users restricted to (allowed domains)** checkbox shown below:
+
+    ![Restrict domains checkbox](static/images/help/restrict-domains-checkbox.png)
```

Comment:
The image link is broken for me. I think saying `static/...` doesn't work here, since it would check for the directory `static` in the folder your doc lies in. You might want to use paths like `/../../static/...` to fix this.